### PR TITLE
Consolidate MTC tutorials 

### PR DESCRIPTION
**Pull request by tiffanyec**
### Description
Consolidated the MoveIt Task Constructor Example and the Pick and Place with MoveIt Task Constructor Tutorial into one. The Pick and Place tutorial was kept and the other example was removed. 

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by stephanie-eng** [comment]
Just a quick note: There are some cross-references from the tutorial to the example page that should be fixed if merging the two!

File: doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst [hunk 1]
```diff
@@ -10,6 +10,46 @@ Pick and Place with MoveIt Task Constructor
 
 This tutorial will walk you through creating a package that plans a pick and place operation using `MoveIt Task Constructor <https://github.com/ros-planning/moveit_task_constructor/tree/ros2/>`_. MoveIt Task Constructor provides a way to plan for tasks that consist of multiple different subtasks (known as stages). This tutorial is intended for those who have a basic understanding of MoveIt and the MoveIt Task Constructor :ref:`concepts <moveit_task_constructor_concepts>`. To learn more about these, read the  :doc:`MoveIt examples </doc/examples/examples>`, including the example page for :doc:`MoveIt Task Constructor </doc/examples/moveit_task_constructor/moveit_task_constructor_tutorial>`.
```
Comment (AndyZe) [inline doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst:11]:
```suggestion
This tutorial will walk you through creating a package that plans a pick and place operation using `MoveIt Task Constructor <https://github.com/ros-planning/moveit_task_constructor/tree/ros2/>`_. MoveIt Task Constructor provides a way to plan for tasks that consist of multiple different subtasks (known as stages).
```

**Review by AndyZe — COMMENTED** [review]
Generally looks good! I found one link that ought to be deleted, though.

**Review by AndyZe — APPROVED** [review]

**Comment by AndyZe** [comment]
Thanks!

**Pull request merged**